cypress/integration/scenarios/with_default_options.spec.ts

```diff
@@ -33,7 +34,21 @@ describe('Menu testing', () => {
     // Check Epinio's side menu
     epinio.checkEpinioNav();
   });
-});
+
+  it('Verify Welcome Screen without Namespaces', () => {
+    cy.clickEpinioMenu('Namespaces');
+    // Deletes all namespaces if detected
+    cy.get("body").then(($body) => {
+      if ($body.text().includes('Delete')) {
+        cy.deleteAllNamespaces()
+      }
+     }
+    )
+    cy.clickEpinioMenu('Applications');
+    cy.get('h1').contains('Welcome to Epinio', { timeout: 4000 }).should('be.visible')
+    cy.createNamespace('workspace')
```

Comment:
Instead of clicking on Namespace, I would use the "Get started" button here, to re-create the workspace namespace. Like that, the button is tested